src/main/java/uk/ac/ebi/ena/webin/cli/validator/manifest/Manifest.java

```diff
@@ -42,6 +42,8 @@ public abstract class Manifest <FileType extends Enum<FileType>> {
     private boolean quick;
     private boolean ignoreErrors;
     private String webinAuthToken;
+    private String webinRestUri;
```

Comment:
I don't think we should do this. Both Webin-cli and webin-sequence-processing should be responsible for having these URLs.